package.json

```diff
@@ -33,7 +33,7 @@
     "npm": ">=3.0.0"
   },
   "dependencies": {
-    "doctest": "0.16.x",
+    "doctest": "0.17.x",
```

Comment:
Would it make sense for us to first make this change in a separate pull request?

Reply:
I think it'll be fine to rearrange the commits and leave it in this PR. Other than adding esm support, there is no pressing reason to update doctest.

Reply:
I force-pushed a new commit order